### PR TITLE
Vak's assorted changes.

### PR DESCRIPTION
**Pull request by Vakothu**
Well, the Glloydstation merge is in the works, might as well open this now.

This PR includes:
- Gives Retired Admins Variable Edit, because we have few enough active admins as is.
- A new away mission, the zombie wild west, originally made by Gale.
  - Fixed Urist Spooky-mob's lacking dead icons.
- Several map tweaks, including, but not limited to
  - Swapping the dorms and lounge, expands the new lounge.
    - Traders now dock at this lounge, not in Cargo.
  - Expands the Janitor closet by a few tiles, replaces the mop bucket with a proper janitorial cart.
  - Adds in a couple windows to the Hatchling's prep area, so you can see if it's docked before opening the door.
  - Adds one of each arcade machine to the Holodeck area, expands windows by 1 so it fits nicely.
  - A couple tweaks to Traders and their outpost.
    - Hooks up the trader's distribution and waste loops to every room, instead of just some.
    - Adds the old Glloydstation Urist vending machines to the trader's outpost.
    - Expands the trader ship's airlock by 1, so it docks nicely with the new lounge dock.
    - Adds in two new trader ships, a seeds/gardener ship, and a stock parts ship.
    - Allows one trader to late-join, if there is not a trader already in-game.
- Recolours the human base sprite to be less yellow-orange, more humanly coloured.
- Adds in Boron as an actual gas, instead of a having canisters of mysterious nothing gas labeled Boron.
- Adds new items to the loadout for Wyrm, these items are 3 accessories, 1 hat, and 1 undersuit:
  - Accessories  
    - The Ubac, in 3 colours.
    - The Dashiki, also in 3 colours.
    - 3 colourable shirts; a turtleneck sweater, a qipao, and a sherwani.
  - Hat
    - The old straw boater/barber hat, by Cheed's request.
  - Undersuit
    - Changed the short skirt undersuit to be colourable, rather then 3 hardcoded colours.
      - Commented out the old uncolourable short skirts, you can make all of them with the new colourable one.
- Allows underwear to be in files other then human.dmi
  - Adds in Hippie's Alien Undershirt, in male and female variants, at Lyeos' request.
- Makes washing machines able to hold a total of 10 items before being full, the parts inside it count as four of these.

### CODE REVIEW

File: code/modules/xgm/boron.dm [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/decl/xgm_gas/hydrogen/boron
```
Comment (Naxodile) [inline code/modules/xgm/boron.dm]: Fairly sure boron doesn't act like hydrogen. In addition this should be in gases.dm

File: code/modules/xgm/boron.dm [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+	id = "boron"
+	name = "Boron-11"
+
+	molar_mass = 0.011	// kg/mol
```
Comment (Naxodile) [inline code/modules/xgm/boron.dm]: Missing specific_heat

File: code/modules/urist/items/clothes/uristloadout.dm [hunk 3]
```diff
@@ -0,0 +1,8 @@
+//Recolourable short skirt, there were 3 hard-coloured ones, and it annoyed me -Vakothu
+
+/obj/item/clothing/under/urist/shortskirt
+	name = "short skirt"
+	desc = "A skirt that barely even reaches your knees."
+	body_parts_covered = LOWER_TORSO|LEGS
```
Comment (Naxodile) [inline code/modules/urist/items/clothes/uristloadout.dm]: Somewhat questionable that a skirt covers your legs.

File: maps/wyrm/wyrm_jobs.dm [hunk 4]
```diff
@@ -65,13 +65,14 @@
 	department = "Civilian"
 	department_flag = CIV
 	faction = "Station"
-	total_positions = 0 //to be opened by admins when desired AT ROUNDSTART ONLY
+	total_positions = 1 //to be opened by admins when desired AT ROUNDSTART ONLY
```
Comment (Naxodile) [inline maps/wyrm/wyrm_jobs.dm]: Comment no longer applies.

File: code/datums/trading/wyrm_traders.dm [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+	possible_trading_items = list(/obj/item/weapon/stock_parts          = TRADER_SUBTYPES_ONLY,
+								/obj/item/stack/cable_coil              = TRADER_SUBTYPES_ONLY,
+								/obj/item/stack/cable_coil/cyborg       = TRADER_BLACKLIST,
+								/obj/item/stack/cable_coil/random       = TRADER_BLACKLIST,
+								/obj/item/stack/cable_coil/cut          = TRADER_BLACKLIST,
+								/obj/item/weapon/cell                   = TRADER_SUBTYPES_ONLY,
+								/obj/item/weapon/cell/mecha             = TRADER_BLACKLIST,
+								/obj/item/weapon/cell/apc               = TRADER_BLACKLIST,
+								/obj/item/weapon/cell/slime             = TRADER_BLACKLIST,
+								/obj/item/weapon/cell/infinite          = TRADER_BLACKLIST,
```
Comment (Naxodile) [inline code/datums/trading/wyrm_traders.dm]: I'm of the opinion traders should be able to acquire things outside of R&D's power level. If need be change the item worth of this to something more accurate of bluespace tech.

File: maps/wyrm/wyrm_shuttles.dm [hunk 6]
```diff
@@ -33,6 +33,7 @@
 
 /area/shuttle/merchant/home
 	name = "\improper Merchant Ship"
+	icon_state = "shuttle"
```
Comment (Naxodile) [inline maps/wyrm/wyrm_shuttles.dm]: Isn't this defined at /area/shuttle?

Comment (Vakothu) [reply maps/wyrm/wyrm_shuttles.dm]: For some reason it wasn't.  It is now.

**Pull request merged**